source/content.css

```diff
@@ -932,3 +932,8 @@ body > .footer li a {
 .mark-all-as-read-confirmed .open-repo-notifications {
 	visibility: hidden;
 }
+
+.rgh-monospace-textareas textarea {
+	font-family: Consolas, 'Liberation Mono', Menlo, Courier, monospace !important;
+	font-size: 13px;
```

Comment:
I used `13px` instead of GitHub's normal `12px` for code blocks because it seemed too small when you're trying to edit it